File: app/src/main/java/com/example/androidbasic/ui/activity/SplashActivity.java
package com.example.androidbasic.ui.activity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.example.androidbasic.R;

/**
 * Created by vikas on 26/9/16.
 */

public class SplashActivity extends BaseActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash_screen);
        launchApplication();
    }

    private void launchApplication() {
        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            public void run() {
                Intent intent = new Intent(SplashActivity.this, HomeActivity.class);
                startActivity(intent);
                finish();
            }
        },3000);
    }
}

File: app/src/main/java/in/vikasrathod/androidbasic/ui/activity/SplashActivity.kt
package `in`.vikasrathod.androidbasic.ui.activity

import `in`.vikasrathod.androidbasic.R
import android.content.Intent
import android.os.Bundle
import android.os.Handler

/**
 * Created by vikas on 26/9/16.
 */

class SplashActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash_screen)
        launchApplication()
    }

    private fun launchApplication() {
        val handler = Handler()
        handler.postDelayed({
            val intent = Intent(this@SplashActivity, LifeCycleAActivity::class.java)
            startActivity(intent)
            finish()
        }, 3000)
    }
}
